File: src/test/java/test/ListTenantExtIdpTest.java
package test;
import cn.authing.sdk.java.client.ManagementClient;
import cn.authing.sdk.java.dto.*;
import cn.authing.sdk.java.model.ManagementClientOptions;
import cn.authing.sdk.java.util.JsonUtils;

import java.util.Collections;



public class ListTenantExtIdpTest {

    private static final String ACCESS_KEY_ID = "AUTHING_ACCESS_KEY_ID";
    private static final String ACCESS_KEY_SECRET = "AUTHING_ACCESS_KEY_SECRET";

    public static void main(String[] args) throws Throwable {
        ManagementClientOptions clientOptions = new ManagementClientOptions();
        clientOptions.setAccessKeyId(ACCESS_KEY_ID);
        clientOptions.setAccessKeySecret(ACCESS_KEY_SECRET);
        ManagementClient client = new ManagementClient(clientOptions);

        ListTenantExtIdpDto request = new ListTenantExtIdpDto();
        request.setTenantId("tenantId_6855");
        request.setAppId("appId_8874");
        request.setType("type_7807");
        request.setPage("page_2737");
        request.setLimit("limit_9343");

        ExtIdpListPaginatedRespDto response = client.listTenantExtIdp(request);
        System.out.println(JsonUtils.serialize(response));
    }

}

File: src/test/go/test/ListTenantExtIdpTest.go
package main

import (
"authing-go-sdk/client"
"authing-go-sdk/dto"
"fmt"
)

func main() {
    options := client.ManagementClientOptions{
        AccessKeyId:     "AUTHING_USERPOOL_ID",
        AccessKeySecret: "AUTHING_USERPOOL_SECRET",
    }
    var err error
    client, err := client.NewClient(&options)
    if err != nil {
        panic(err)
    }
    request := dto.ListTenantExtIdpDto{
        TenantId: "tenantId_6024",
        AppId: "appId_5221",
        Type: "type_1026",
        Page: "page_3707",
        Limit: "limit_7452",
    }
    response := client.ListTenantExtIdp(&request)
    fmt.Println(response)
}

